docs/source/reference/indexing.rst

```diff
@@ -24,6 +24,7 @@ Properties
    Index.ndim
    Index.empty
    Index.T
+   Index.shape
```

Comment:
oh i'm very sorry for bothering you again,

but how about we move this to right before `Index.name`?

because we should mimic pandas' as much as possible not only for functionality, but also docs. 😄 

![스크린샷 2019-11-30 오후 4 41 36](https://user-images.githubusercontent.com/44108233/69897520-81ac6a00-1390-11ea-9a6c-b5d5814fa66d.png)

Reply:
Sorry to miss it, I'll pay attention to it next time.

Reply:
you don't have to apology for this. :)

It was really common to me too haha.